src/main/java/frc/robot/subsystems/drive/PhoenixDrive.java

```diff
@@ -204,46 +198,67 @@ public void applyGoalSpeeds() {
         Logger.recordOutput("Drive/idling", idling);
 
         // sticks zeroed out will cause false idle when wanting alignment
-        if (idling && !aligning) {
-            request = new SwerveRequest.Idle();
-        } else if (fieldCentric) {
-            if (aligning) {
-                SwerveRequest.FieldCentricFacingAngle alignRequest =
-                        new SwerveRequest.FieldCentricFacingAngle()
-                                .withVelocityX(goalSpeeds.vxMetersPerSecond)
-                                .withVelocityY(goalSpeeds.vyMetersPerSecond)
-                                .withTargetDirection(this.getAlignment().get())
-                                .withDeadband(0.0)
-                                .withRotationalDeadband(0.0)
-                                .withDriveRequestType(DriveRequestType.Velocity);
-
-                alignRequest.HeadingController.setPID(
-                        PhoenixDriveConstants.alignmentkP,
-                        PhoenixDriveConstants.alignmentkI,
-                        PhoenixDriveConstants.alignmentkD);
-
-                request = alignRequest;
+        if (DriverStation.isTeleop()) {
```

Comment:
We will need to fix this after competition, but architecturally the code between autonomous and teleop should be shared. There shouldn't be a difference in operation between the two other than the joysticks commanding the drivetrain rather than path planner

Reply:
yeah, i can fix that real quick now that i think about it, that was just a super quick fix. @jkleiber do you want me to add that quickly to this branch or just save for after comp so theres no breaking changes

Reply:
@linglejack06 let's open a ticket to fix this after competition